tags-config-service-impl/src/main/java/org/hypertrace/tag/config/service/TagConfigServiceImpl.java

```diff
@@ -23,9 +28,20 @@
 @Slf4j
 public class TagConfigServiceImpl extends TagConfigServiceGrpc.TagConfigServiceImplBase {
   private final ConfigServiceCoordinator configServiceCoordinator;
+  private final List<String> SYSTEM_TAGS;
+  private final List<String> SYSTEM_TAG_PATHS =
```

Comment:
Please don't define them explicitly. You need to consume whatever tags are defined in config.

Assume that system tags are defined as array of tags under `tag.config.service` as `system.tags`. You need to read them as array and use them.

Reply:
Will make this change.

Reply:
Done.